gcs/blockcf/blockcf.go

```diff
@@ -123,8 +123,9 @@ func Regular(block *wire.MsgBlock) (*gcs.FilterV1, error) {
 	// byte of the script and removing it allows users of the filter to only
 	// match against a normal P2PKH or P2SH script, instead of many extra
 	// matches for each tag.
+	isTreasuryEnabled := false // XX is this ok?
```

Comment:
Yes, this is accurate.  v1 filters are deprecated and, while they are non consensus (unlike v2) they really shouldn't be changed without a version bump.

Reply:
Fix committed.